docs/source/user_guide/timeseries.ipynb

```diff
@@ -293,10 +293,15 @@
     "You'll notice that in the code snippets here, we use the ``predict_in_sample`` pipeline method as opposed to the usual ``predict`` method. What's the difference?\n",
     "\n",
     "* ``predict_in_sample`` is used when the target value is known on the dates we are predicting on. This is true in cross validation. This method has an expected ``y`` parameter so that we can compute features using previous target values for all of the observations on the holdout set.\n",
-    "* ``predict`` is used when the target value is not known, e.g. the test dataset. The y parameter is not expected as only the target is observed in the training set. The test dataset must be separated by ``gap`` days from the training dataset. For the moment, the test set size be less than or equal to ``forecast_horizon``.\n",
-    "\n",
-    "Here is an example of these two methods in action:\n",
+    "* ``predict`` is used when the target value is not known, e.g. the test dataset. The y parameter is not expected as only the target is observed in the training set. The test dataset must be separated by ``gap`` units from the training dataset. For the moment, the test set size be less than or equal to ``forecast_horizon``.\n",
```

Comment:
"For the moment, the test set size be less than or equal to ``forecast_horizon``" ----> "For the moment, the test set size **must** be less than or equal to ``forecast_horizon``"?